rails_docker/Gemfile.txt

```diff
@@ -52,19 +52,19 @@ group :development, :test do
   gem 'guard-rspec' # Auto-run specs
   gem 'shoulda-matchers' # Tests common Rails functionalities
   gem 'capybara' # Integration testing
-  gem 'poltergeist' # Headless browser
-  gem 'selenium-webdriver', '2.53.4' # Ruby bindings for Selenium/WebDriver
+  gem 'selenium-webdriver', '3.7.0' # Ruby bindings for Selenium/WebDriver
   gem 'database_cleaner' # Use Database Cleaner
 
   gem 'byebug' # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'spring' # Spring speeds up development by keeping your application running in the background.
   gem 'spring-commands-rspec' # This gem implements the rspec command for Spring.
-  gem 'spring-watcher-listen', '~> 2.0.0' # Makes Spring watch the filesystem for changes using Listen
-  gem 'listen', '~> 3.0.5' # Listens to file modifications
+  gem 'spring-watcher-listen', '2.0.1' # Makes Spring watch the filesystem for changes using Listen
+  gem 'listen', '3.1.5' # Listens to file modifications
   gem 'letter_opener' # Preview mail in the browser instead of sending.
 end
 
 group :test do
+  gem 'webmock' # Library for stubbing and setting expectations on HTTP requests in Ruby
```

Comment:
@anduonghien any reason to add this, considering we already have `vcr`

Reply:
@abhinavmsra `webmock` is a dependency of `vcr`. But `webmock` allows you to do things outside of `vcr` e.g. custom Mock, using `stub_request`